WeakAuras/Prototypes.lua

```diff
@@ -6638,6 +6638,7 @@ Private.event_prototypes = {
     type = "item",
     events = {
       ["events"] = {
+        "BAG_UPDATE_COOLDOWN",
```

Comment:
But here I think we need that batched treatement. Though BAG_UPDATE_COOLDOWN fires a lot with no discernable reason. I think we should also only do that if we are tracking charges, as I think everything else is handled by BAG_UPDATE_DELAYED. At least in my testing that BAG_UPDATE_DELAYED fires for everything I could think of.